### PR TITLE
Update style.css

### PR DESCRIPTION
**Pull request by elic4vet**
I added some styles on the hero section, i will continue working on this subject later. Hope it's okay for you :)

### CODE REVIEW

**Comment by elic4vet** [comment]
I had to repost my pull request because i couldn't find my changes concerning the html file.  Hope it's okay for you :)

**Comment by elic4vet** [comment]
i created the about us section and added the corresponding link that navigates from home to the about section and back to home

File: css/style.css [hunk 1]
```diff
@@ -190,6 +190,35 @@ ul {
...
+  display: flex;
+  justify-content: center;
+  color: var(--text-color-light);
+  padding: 100px 0;
+}
+.hero-title {
+  font-family: "Roboto-Black", sans-serif;
+  font-size: 2.25rem;
+  margin-bottom: 10px;
+}
```
Comment (basiacarvalho) [inline css/style.css:204]: Decide whether you want to always leave one line space between style declarations or have it all one by one. I read on the internet that is better to keep one empty line in style for readability (so I always separate style declarations with one line).

File: index.html [hunk 2]
```diff
@@ -24,10 +24,9 @@
     </header>
     <main>
       <!--1st section - Hero -->
-      <section class="section">
+      <section class="hero">
```
Comment (basiacarvalho) [inline index.html:27]: You can leave the common class "section" in the section tag, and apart from that add the second class name "hero" and put in "hero" only what varies from the normal "section" styling.

**Review by basiacarvalho — APPROVED** [review]
Hi, I left 2 comments :)

**Comment by elic4vet** [comment]
I added the course link on the about us section

**Comment by elic4vet** [comment]
•	I added an image to the hero section.
•	I also added an image container to better divide and organize these two sections.
•	I added styles for the two divs and the image.

- [ ] 	I need to work later on the style of the page and @media queries for the mob + tablet version.

File: index.html [hunk 3]
```diff
@@ -16,24 +16,35 @@
...
             Quisquam, voluptatum. Quisquam, voluptatum. Quisquam, voluptatum.
           </p>
           <a class="hero-btn" href="#">Signup</a>
         </div>
+
+          <div class="image-container">
+            <img 
+              class="hero-img" 
+              src="./images/heroimg.jpg" 
+              alt="a person sitting in front of a computer"
```
Comment (basiacarvalho) [inline index.html:41]: I would probably write alt="an open laptop with code display on a desk with a plant and a mug" for the hero img

**Review by basiacarvalho — APPROVED** [review]

**Review by basiacarvalho — CHANGES_REQUESTED** [review]
Hi, 

One thing to fix - look at my proposition of an "alt" for hero img :) 
"an open laptop with code displayed on a desk with a plant and a mug"

**Comment by elic4vet** [comment]
I changed the text description of the alt as suggested before

**Review by basiacarvalho — APPROVED** [review]

**Pull request merged**